pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -144,6 +146,8 @@ public BaseBrokerRequestHandler(PinotConfiguration config, RoutingManager routin
     _defaultHllLog2m = _config.getProperty(CommonConstants.Helix.DEFAULT_HYPERLOGLOG_LOG2M_KEY,
         CommonConstants.Helix.DEFAULT_HYPERLOGLOG_LOG2M);
     _enableQueryLimitOverride = _config.getProperty(Broker.CONFIG_OF_ENABLE_QUERY_LIMIT_OVERRIDE, false);
+    _enableSegmentPartitionedDistinctCountOverride =
```

Comment:
Can this be query flag instead of config? Issue with config is that we need to then enforce the config is set on all instances.

Reply:
This can be. a helix based cluster config now :p

Reply:
Added queryoption to allow query time config as well.